05_test.sh

```diff
@@ -11,6 +11,12 @@ export ACTION="ci_test_provision"
 
 "${METAL3_DIR}"/scripts/run.sh
 
+# wait until status of Metal3Machine is rebuilt
+while [ -z "${status}" ]
+do
+    status=$(kubectl get m3m -n "${NAMESPACE}" -o=jsonpath="{.items[*]['status.ready']}")
```

Comment:
Don't we need to sleep sometime in between retries ?

Reply:
yep that could be easier, but I didn't know what would be the best value for the sleep

Reply:
I know that controller is fast enough to rebuilt the status, perhaps ~10-15 secs would be enough ?, not sure

Reply:
the problem with this approach is, it relies only on m3m, and what if there is another object that is waiting for its status to be rebuilt too. Then this won't be enough. But controller does reconciliation in parallel though :)

Reply:
then probably sleep for 1s after retry should be ok

Reply:
agree